package.json

```diff
@@ -30,6 +30,8 @@
       "lerna exec --concurrency 1 --stream --bail=false -- npm run test:insiders",
     "test:unit":
       "lerna exec --concurrency 1 --stream --bail=false -- npm run test:unit",
+    "test:vscode-integration":
```

Comment:
I added a new test category: vscode-integration for tests that were previously considered 'unit' tests, but are now considered 'integration' tests since they depend on a running vscode instance.

Reply:
The test category `vscode-integration` looks good to me. Since these tests depend on a running vscode instance, I believe we actually have two scripts in most of the "salesforcedx-vscode-*" `package.json`s that would fall into this category: `test:insiders:` and the `test:unit` that was renamed. Would it be helpful to rename the `test:insiders` script to also reflect this new category?

Reply:
@AnanyaJha, that's a good point. What do you think of renaming the `test:insiders` category to `test:insiders-integration`?

Reply:
Yeah, I think that sounds good!